### PR TITLE
amber mbar bace example

### LINKED ISSUE #20: AMBER FEP dataset
Add a FEP/MBAR data set for AMBER.

To be used with PR alchemistry/alchemlyb#51

### PR DESCRIPTION
**Pull request by brycestx**
Fixes #20 

This PR adds a simple JACS dataset perturbation from Bace. Amber .out files are supplied for both complex and solvated systems as well as full decharge > vdw > recharge legs of the free energy protocol.

### CODE REVIEW

File: src/alchemtest/amber/access.py [hunk 1]
```diff
@@ -7,6 +7,37 @@
...
+    -------
+    data: Bunch
+        Dictionary-like object, the interesting attributes are:
+
+        - 'data' : the data files files by system and alchemical leg
+
+    """
+    module_path = dirname(__file__)
+    data = {'complex':
+                {'decharge': glob(join(module_path, 'bace_CAT-13d~CAT-17a/complex/decharge/*/ti-*.out')),
```
Comment (orbeckst) [inline src/alchemtest/amber/access.py]: compress the files; bz2 is probably best

**Comment by brycestx** [comment]
@orbeckst I think I addressed your changes

File: src/alchemtest/amber/access.py (hunk 1 above)
Comment (orbeckst) [reply src/alchemtest/amber/access.py]: You need to have your paths as `ti-*.out.bz2`

File: src/alchemtest/amber/access.py [hunk 2]
```diff
@@ -7,6 +7,37 @@
...
+                 'recharge': glob(join(module_path, 'bace_CAT-13d~CAT-17a/complex/recharge/*/ti-*.out')),
+                 'vdw': glob(join(module_path, 'bace_CAT-13d~CAT-17a/complex/vdw/*/ti-*.out'))
+                 },
+            'solvated':
+                {'decharge': glob(join(module_path, 'bace_CAT-13d~CAT-17a/solvated/decharge/*/ti-*.out')),
```
Comment (orbeckst) [inline src/alchemtest/amber/access.py]: You need to have your paths as `ti-*.out.bz2`

**Comment by orbeckst** [comment]
@brycestx I will quickly finish the PR and merge.

**Comment by brycestx** [comment]
Awesome, thanks @orbeckst

**Pull request merged**